partiql-ast/src/main/java/org/partiql/ast/SelectItem.java

```diff
@@ -71,5 +73,15 @@ public List<AstNode> getChildren() {
         public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
             return visitor.visitSelectItemExpr(this, ctx);
         }
+
+        @NotNull
+        public org.partiql.ast.expr.Expr getExpr() {
+            return this.expr;
+        }
+
+        @Nullable
+        public Identifier getAsAlias() {
+            return this.asAlias;
+        }
```

Comment:
Is there a reason you didn't use Lombok here?

Reply:
No reason. Think I was just testing out the de-lomboked code here. I'll change to use Lombok